osquery/events/kernel.cpp

```diff
@@ -51,11 +51,13 @@ Status KernelEventPublisher::setUp() {
     queue_ = new CQueue(kKernelDevice, kKernelQueueSize);
   } catch (const CQueueException &e) {
     queue_ = nullptr;
+    LOG(WARNING) << "Error initializing CQueue object: " << e.what();
```

Comment:
The `CQueue` is a bit opaque to users, we should say something like "Error initializing kernel structures: "